.github/Style Guide.md

```diff
@@ -0,0 +1,227 @@
+
+To enable GitHub linking, an @ is placed before the username:
+<br>✅ `... - thanks to @peterohanrahan (Peter O'Hanraha-Hanrahan)`
+<br>❌ `... - thanks to peterohanrahan (Peter O'Hanraha-Hanrahan)`
+
+People may contribute anonymously, in which case the name and brackets are omitted:
+<br>✅`... - thanks to @anonymousman`
+<br>❌`... - thanks to @anonymousman ()`
+<br>❌`... - thanks to @anonymousman (Anonymous)`
+However, maintainers are grateful if a note of this is made in the PR so that they don't then double-check whether it is an omission or genuine.
+
+Where there are multiple contributors to a PR, they should be laid out individually, using the Oxford comma where there are more than two:
+<br>✅`... - thanks to @KieraN (Kiera Nightly), @GWall (Greg Wallets), and @cushypete (Peter Cushy)`
+<br>❌`... - thanks to @KieraN, @GWall and @cushypete (Kiera Nightly, Greg Wallets, and Peter Cushy)`
+
+Where a contribution has been made on behalf of someone without a GitHub account, then the GH contributor will be written in the normal way, and the non-GH contributor simply written as a name.
+<br>✅`... - thanks to @UOneMc (Ewan McGreg) and James Jones`
+<br>❌`... - thanks to @UOneMc (Ewan McGreg) and (James Jones)`
+<br>❌`... - thanks to @UOneMc (Ewan McGreg)`
+
+Generally, contributors are simply ordered in any order, although if one is demanded it should be in alphabetical order by first name.
+Additionally, contributors are regarded equally - the changelog should simply list the names and nothing more. 
+Reviewers should not ask to be added to the changelog simply for either adding comments, making suggestions or for making minor edits to the PR. Where a reviewer goes beyond this and effectively 'takes over' a PR, then they may add themselves.
+
+## SF/aviation specific terms <a name="terms"></a>
+**Abbreviations**:
+The following are abbreviations and should always be written as such in capitals. Their full definitions are given here for completeness but should not be written in full. If plural, a lowercase `s` is added.
+
+- `AC` (Area Control)
+- `APP` (Approach, referring to the position)
+- `GND` (Ground, referring to the position)
+- `GMC` (Ground Movement Control)
+- `GMP` (Ground Movement Planner)
+- `DEL` (Delivery)
+- `ICAO` (International Civil Aviation Organization, but used usually to mean an ICAO code relating to an airport)
+- `NDB` (Non-Directional Beacon)
+- `RMA` (Radar Maneuvering Area)
+- `ScAC` (Scottish Area Control)
+- `SDR` (Standard Departure Route)
+- `SID` (Standard Instrument Departure)
+- `SMR` (Surface Movement Radar, but refers to the map upon which it is drawn)
+- `STAR` (Standard Terminal Arrival Route)
+- `STC` (Scottish Terminal Control)
+- `TC` (Terminal Control)
+- `TWR` (Tower, referring to the position)
+- `VOR` (VHF Omnidirectional Beacon)
+
+**Agreements**:
+Agreements typically use the notation `->` to indicate the direction of travel. For agreement in both directions these use `<->`.

```

Comment:
Is this referring to COPX?

Reply:
Yes - I'll add that in

Reply:
OK, in which case, for climbing agreements it's `^310` or for descending, it's `|160`. For level by agreements, it's without either.

Reply:
This was meant for COPX related entries in titles, changelogs, etc. but that's a good point, so I'll document that too

Reply:
@hsugden is there a reason that the symbol precedes the fixes but the number is on the end? I don't plan to change this as it would require changing a lot but interested to know if there's a reasoning behind it

Reply:
Oh, I see, maybe that isn’t style guide then this is more for the wiki! My bad. Didn’t completely follow it seems 😄 

‘descending via VATON’

‘TNT-25’ = ‘25 before TNT’

But that’s me coming up with a post factum explanation to fit the convention!